source/v4/users.yaml

```diff
@@ -551,4 +551,33 @@
         '401':
           $ref: '#/responses/Unauthorized'
         '403':
-          $ref: '#/responses/Forbidden'
\ No newline at end of file
+          $ref: '#/responses/Forbidden'
+
+  /users/{user_id}/audits:
+    get:
+      tags:
+        - users
+      summary: Get user's audits
+      description: |
+        Get a list of audit by providing the user GUID.
+        ##### Permissions
+        Must be logged in as the user being updated or have the `edit_other_users` permission.
```

Comment:
I think there's a copy/paste error here, can you change this to "Must be logged in as the user or have the `edit_other_users` permission."